extensions/cornerstone/src/utils/measurementServiceMappings/Length.js

```diff
@@ -57,12 +61,22 @@ const Length = {
       StudyInstanceUID,
     } = getSOPInstanceAttributes(element);
 
+    const displaySets = DisplaySetService.getDisplaySetsForSeries(
+      SeriesInstanceUID
+    );
+    const displaySet = displaySets.find(ds => {
+      return (
+        ds.images && ds.images.some(i => i.SOPInstanceUID === SOPInstanceUID)
+      );
+    });
```

Comment:
We sure do this a lot

Reply:
`DisplaySetService.getDisplaySetForImage(SOPInstanceUID, SeriesInstanceUID?)`

Where second option, if provided, is more efficient?